examples/bri-3/src/bri/auth/capabilities/login/login.command.handler.ts

```diff
@@ -11,25 +11,23 @@ export class LoginCommandHandler implements ICommandHandler<LoginCommand> {
     const { message, signature, publicKey } = command;
 
     const bpiSubject = await this.authAgent.getBpiSubjectByPublicKey(publicKey);
-    if (
-      bpiSubject &&
-      bpiSubject.publicKey === publicKey &&
-      bpiSubject.loginNonce === message
-    ) {
-      if (this.authAgent.verify(message, signature, publicKey)) {
-        const bpiSubject = await this.authAgent.getBpiSubjectByPublicKey(
-          publicKey,
-        );
-        const payload = {
-          username: bpiSubject.name,
-          sub: bpiSubject.id,
-        };
-        bpiSubject.loginNonce = '';
```

Comment:
i changed this so new nonce is simply generated for every login

Reply:
not sure if we shall allow a relogin attempt if a user is already authenticated.